examples/multi-region/versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 4.4"
```

Was this version bump accidentally missed?
```suggestion
      version = ">= 6.0"
```

---

Yep, I think so. It will also need to the README to be updated. (I'll try to do that tomorrow when I'm back on the laptop with the tools available) (the other examples won't need that, since they won't use the region parameter (the module will bump up the required version that Terraform actually installs though)

---

Updated that one. Left the others - they don't use the new property and would be fine with older versions of the module using older versions of the provider.